src/lib.c

```diff
@@ -59,11 +59,18 @@ PyMODINIT_FUNC PyInit_lib(void) {
   import_array();
   import_umath();
 
+  /* add quotes to GEOS_VERSION_PATCH to handle (e.g.) 0beta1, then take the first digit */
```

Comment:
Can you maybe refer to the gcc docs (e.g. https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html)?

Reply:
Sure, although I opted to use more current https://gcc.gnu.org/onlinedocs/cpp/Stringizing.html

It seems they have been shifting their term from "stringification" to "stringizing".